charts/sonarqube-dce/values.schema.json

```diff
@@ -48,15 +48,47 @@
             }
         },
         "account": {
+            "type": "object",
+            "deprecated": true,
+            "$comment": "(DEPRECATED) Please use `setAdminPassword` with lowercase a instead"
```

Comment:
i dont understand the "lowercase a" sentence.

Reply:
Indeed, this is a copy-and-paste issue...